ember-cli-build.js

```diff
@@ -4,7 +4,10 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   var app = new EmberAddon(defaults, {
     babel: {
-      plugins: ['transform-decorators-legacy']
+      plugins: [
+        'transform-decorators-legacy',
+        'transform-class-properties'
```

Comment:
Added this for the new test that checks that the error is thrown if `@action` is applied to a non-function. I'll be submitting another PR to update the README and include tests/examples for ES6 classes soon.